colabs/wandb-model-registry/New_Model_Logging_in_W&B.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>nit but I prefer import wandb and wandb.login() at top level cells as opposed to lower</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>